src/Sts/RegionalEndpoints/Configuration.php

```diff
@@ -4,10 +4,12 @@
 class Configuration implements ConfigurationInterface
 {
     private $endpointsType;
+    private $isDefault;
```

Comment:
isDefault what?  This variable name should be more explanatory, as of right now I'm not sure what it does

Reply:
Basically just the default value or the 'fallback' we use.  I think the latter might make more sense as a variable name